encoder/libav_encoder.cpp

```diff
@@ -411,13 +478,15 @@ void LibAvEncoder::initOutput()
 
 void LibAvEncoder::deinitOutput()
 {
+
 	if (!out_fmt_ctx_)
 		return;
 
 	av_write_trailer(out_fmt_ctx_);
 
 	if (!(out_fmt_ctx_->flags & AVFMT_NOFILE))
 		avio_closep(&out_fmt_ctx_->pb);
+	out_fmt_ctx_->pb = nullptr;
```

Comment:
I *think* this might not be needed.  But please do check.

Reply:
I guess the conclusion is that this _is_ required?